context/app/static/js/pages/Dataset/Dataset.jsx

```diff
@@ -75,8 +75,12 @@ function DatasetDetail(props) {
   const allCollections = useCollectionsData(elasticsearchEndpoint, nexusToken);
   const collectionsData = getCollectionsWhichContainDataset(uuid, allCollections);
 
+  const shouldDisplayVisualization = vitData && ('name' in vitData || (vitData[0] && 'name' in vitData[0]));
+  if (vitData && !shouldDisplayVisualization) {
+    throw Error('Vitessce config does not have expected structure');
```

Comment:
No, I don't think that's necessary. I hit this working on this PR, because I didn't realize the `name` was required: it's probably going to be an error in development, rather than in production.

If we did want to catch this in production, I think we should use an error boundary in react, rather than a special Vitessce conf... actually, that's a good point: The error-message vitessce config is just an awkward way to communicate between layers: I'll simplify the response in the python code, and put an error boundary just around vitessce, and that will handle all cases.

I'll ask for re-review in a bit. Thanks for prodding me.